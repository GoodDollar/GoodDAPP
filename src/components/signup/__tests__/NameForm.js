import React from 'react'
import renderer from 'react-test-renderer'
import { withThemeProvider } from '../../../__tests__/__util__'
import { getWebRouterComponentWithMocks } from './__util__'

describe('NameForm', () => {
  it('matches snapshot', async () => {
    const NameForm = withThemeProvider(
      getWebRouterComponentWithMocks('../NameForm', {
        fullName: 'Kevin Bardi',
      }),
    )
    let component

    // eslint-disable-next-line require-await
    await renderer.act(async () => (component = renderer.create(<NameForm />)))
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('matches snapshot with empty value', async () => {
    const NameForm = withThemeProvider(
      getWebRouterComponentWithMocks('../NameForm', {
        fullName: '',
      }),
    )
    let component

    // eslint-disable-next-line require-await
    await renderer.act(async () => (component = renderer.create(<NameForm />)))
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
