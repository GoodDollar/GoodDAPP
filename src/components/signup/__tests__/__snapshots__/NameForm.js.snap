// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NameForm matches snapshot 1`] = `
<div
  className="css-view-1dbjc4n r-flex-13awgt0 r-pointerEvents-12vffkv"
>
  <div
    className="css-view-1dbjc4n r-flex-13awgt0 r-paddingBottom-97e31f r-paddingLeft-bv2aro r-paddingRight-hxflta r-paddingTop-10xqauy"
  >
    <div
      className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-display-6koalj r-flexDirection-eqz5dr r-flexGrow-16y2uox r-maxHeight-87ditf r-overflowX-lltvgl r-overflowY-1rnoaur r-padding-nsbfu8 r-position-bnwqim r-width-13qz1uu"
      data-name="viewWrapper"
    >
      <div
        className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-borderRadius-1f0042m r-flexGrow-16y2uox r-padding-t60dpp r-paddingHorizontal-1e081e0 r-paddingVertical-1yzf0co"
      >
        <div
          className="css-view-1dbjc4n r-alignItems-1oszu61 r-flexDirection-eqz5dr r-flexGrow-16y2uox"
          style={
            Object {
              "WebkitBoxPack": "space-evenly",
              "WebkitJustifyContent": "space-evenly",
              "justifyContent": "space-evenly",
              "msFlexPack": "space-evenly",
            }
          }
        >
          <div
            className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-borderRadius-1f0042m r-flexGrow-16y2uox r-paddingHorizontal-1e081e0 r-paddingVertical-1yzf0co"
            style={
              Object {
                "WebkitBoxPack": "start",
                "WebkitJustifyContent": "flex-start",
                "justifyContent": "flex-start",
                "msFlexPack": "start",
              }
            }
          >
            <div
              className="css-view-1dbjc4n r-alignItems-1oszu61 r-flexDirection-eqz5dr r-height-4d76ec r-minHeight-ql8eny"
              style={
                Object {
                  "WebkitBoxPack": "start",
                  "WebkitJustifyContent": "flex-start",
                  "justifyContent": "flex-start",
                  "msFlexPack": "start",
                }
              }
            >
              <div
                className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz r-justifyContent-1777fci r-marginVertical-1r5su4o"
              >
                <div
                  className="css-text-901oao r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-evnaw r-fontWeight-majxgm r-lineHeight-17rnw9f r-marginVertical-c8eef1 r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-3twk1y"
                  dir="auto"
                >
                  Hi, nice to meet you.
 Please enter your full name
                </div>
              </div>
              <div
                className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz r-justifyContent-1777fci r-marginBottom-5oul0u r-marginTop-1t01tom"
              >
                <div
                  className="css-view-1dbjc4n r-marginBottom-5oul0u r-width-13qz1uu"
                >
                  <div
                    className="css-view-1dbjc4n r-marginBottom-5oul0u r-width-13qz1uu"
                  >
                    <input
                      autoCapitalize="sentences"
                      autoComplete="on"
                      autoCorrect="on"
                      autoFocus={true}
                      className="css-textinput-1cwyjr8 r-backgroundColor-1niwhzg r-borderBottomColor-1ny7kx0 r-borderBottomWidth-qklmqi r-color-cddv6f r-fontFamily-6jnb14 r-fontSize-1i10wst r-paddingHorizontal-1dye5f7 r-paddingVertical-oyd9sg r-placeholderTextColor-1xktb6b r-textAlign-q4m81j"
                      data-focusable={true}
                      dir="auto"
                      id="Component_input"
                      onBlur={[Function]}
                      onChange={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      onKeyPress={[Function]}
                      onSelect={[Function]}
                      onTouchStart={[Function]}
                      readOnly={false}
                      spellCheck={true}
                      type="text"
                      value="Kevin Bardi"
                    />
                  </div>
                  <div
                    className="css-text-901oao r-paddingHorizontal-1e081e0 r-paddingVertical-oyd9sg r-textAlign-q4m81j"
                    dir="auto"
                    style={
                      Object {
                        "WebkitTransform": "translateY(0px)",
                        "color": "rgba(250,108,119,1.00)",
                        "direction": "ltr",
                        "fontFamily": "Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                        "fontSize": "12px",
                        "fontWeight": "400",
                        "height": "18px",
                        "opacity": 0,
                        "paddingBottom": "0px",
                        "paddingLeft": "0px",
                        "paddingTop": "0px",
                        "transform": "translateY(0px)",
                      }
                    }
                  >
                    
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz"
          style={
            Object {
              "WebkitBoxPack": "end",
              "WebkitJustifyContent": "flex-end",
              "justifyContent": "flex-end",
              "msFlexPack": "end",
            }
          }
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1oszu61 r-display-6koalj r-flex-13awgt0 r-justifyContent-1777fci r-padding-t60dpp"
            style={
              Object {
                "backgroundColor": "rgba(0,175,255,1.00)",
                "borderBottomColor": "rgba(0,175,255,1.00)",
                "borderBottomLeftRadius": "50px",
                "borderBottomRightRadius": "50px",
                "borderBottomStyle": "solid",
                "borderBottomWidth": "0px",
                "borderLeftColor": "rgba(0,175,255,1.00)",
                "borderLeftStyle": "solid",
                "borderLeftWidth": "0px",
                "borderRightColor": "rgba(0,175,255,1.00)",
                "borderRightStyle": "solid",
                "borderRightWidth": "0px",
                "borderTopColor": "rgba(0,175,255,1.00)",
                "borderTopLeftRadius": "50px",
                "borderTopRightRadius": "50px",
                "borderTopStyle": "solid",
                "borderTopWidth": "0px",
                "boxShadow": "0px 0.75px 1.5px rgba(0,0,0,0.24)",
                "marginBottom": "0px",
                "marginLeft": "0px",
                "marginRight": "0px",
                "marginTop": "0px",
                "minHeight": "44px",
                "minWidth": "64px",
                "paddingLeft": "0px",
                "paddingRight": "0px",
              }
            }
          >
            <div
              className="css-cursor-18t94o4 css-view-1dbjc4n r-cursor-1loqt21 r-flex-13awgt0 r-overflow-1udh08x r-position-bnwqim r-touchAction-1otgn73"
              data-focusable={true}
              disabled={false}
              onKeyDown={[Function]}
              onKeyPress={[Function]}
              onKeyUp={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              role="button"
              style={
                Object {
                  "WebkitAlignSelf": "stretch",
                  "alignSelf": "stretch",
                  "borderBottomLeftRadius": "50px",
                  "borderBottomRightRadius": "50px",
                  "borderTopLeftRadius": "50px",
                  "borderTopRightRadius": "50px",
                  "msFlexItemAlign": "stretch",
                }
              }
              tabIndex="0"
            >
              <div
                className="css-view-1dbjc4n r-alignItems-1awozwy r-flex-13awgt0 r-flexDirection-18u37iz r-justifyContent-1777fci"
                style={
                  Object {
                    "WebkitAlignSelf": "stretch",
                    "alignSelf": "stretch",
                    "msFlexItemAlign": "stretch",
                  }
                }
              >
                <div
                  className="css-view-1dbjc4n r-marginHorizontal-1ydw1k6 r-marginVertical-712mwr"
                >
                  <div
                    className="css-view-1dbjc4n r-alignItems-1awozwy r-display-6koalj r-flex-13awgt0 r-flexDirection-18u37iz"
                  >
                    <div
                      className="css-text-901oao r-alignItems-1awozwy r-color-jwli3a r-fontFamily-vnw8o6 r-fontSize-ubezar r-fontWeight-majxgm r-justifyContent-1777fci r-letterSpacing-oxtfae r-lineHeight-hbpseb r-paddingTop-njp1lv r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-tsynxw"
                      dir="auto"
                    >
                      Next
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`NameForm matches snapshot with empty value 1`] = `
<div
  className="css-view-1dbjc4n r-flex-13awgt0 r-pointerEvents-12vffkv"
>
  <div
    className="css-view-1dbjc4n r-flex-13awgt0 r-paddingBottom-97e31f r-paddingLeft-bv2aro r-paddingRight-hxflta r-paddingTop-10xqauy"
  >
    <div
      className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-display-6koalj r-flexDirection-eqz5dr r-flexGrow-16y2uox r-maxHeight-87ditf r-overflowX-lltvgl r-overflowY-1rnoaur r-padding-nsbfu8 r-position-bnwqim r-width-13qz1uu"
      data-name="viewWrapper"
    >
      <div
        className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-borderRadius-1f0042m r-flexGrow-16y2uox r-padding-t60dpp r-paddingHorizontal-1e081e0 r-paddingVertical-1yzf0co"
      >
        <div
          className="css-view-1dbjc4n r-alignItems-1oszu61 r-flexDirection-eqz5dr r-flexGrow-16y2uox"
          style={
            Object {
              "WebkitBoxPack": "space-evenly",
              "WebkitJustifyContent": "space-evenly",
              "justifyContent": "space-evenly",
              "msFlexPack": "space-evenly",
            }
          }
        >
          <div
            className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-borderRadius-1f0042m r-flexGrow-16y2uox r-paddingHorizontal-1e081e0 r-paddingVertical-1yzf0co"
            style={
              Object {
                "WebkitBoxPack": "start",
                "WebkitJustifyContent": "flex-start",
                "justifyContent": "flex-start",
                "msFlexPack": "start",
              }
            }
          >
            <div
              className="css-view-1dbjc4n r-alignItems-1oszu61 r-flexDirection-eqz5dr r-height-4d76ec r-minHeight-ql8eny"
              style={
                Object {
                  "WebkitBoxPack": "start",
                  "WebkitJustifyContent": "flex-start",
                  "justifyContent": "flex-start",
                  "msFlexPack": "start",
                }
              }
            >
              <div
                className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz r-justifyContent-1777fci r-marginVertical-1r5su4o"
              >
                <div
                  className="css-text-901oao r-color-cddv6f r-fontFamily-vnw8o6 r-fontSize-evnaw r-fontWeight-majxgm r-lineHeight-17rnw9f r-marginVertical-c8eef1 r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-3twk1y"
                  dir="auto"
                >
                  Hi, nice to meet you.
 Please enter your full name
                </div>
              </div>
              <div
                className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz r-justifyContent-1777fci r-marginBottom-5oul0u r-marginTop-1t01tom"
              >
                <div
                  className="css-view-1dbjc4n r-marginBottom-5oul0u r-width-13qz1uu"
                >
                  <div
                    className="css-view-1dbjc4n r-marginBottom-5oul0u r-width-13qz1uu"
                  >
                    <input
                      autoCapitalize="sentences"
                      autoComplete="on"
                      autoCorrect="on"
                      autoFocus={true}
                      className="css-textinput-1cwyjr8 r-backgroundColor-1niwhzg r-borderBottomColor-1ny7kx0 r-borderBottomWidth-qklmqi r-color-cddv6f r-fontFamily-6jnb14 r-fontSize-1i10wst r-paddingHorizontal-1dye5f7 r-paddingVertical-oyd9sg r-placeholderTextColor-1xktb6b r-textAlign-q4m81j"
                      data-focusable={true}
                      dir="auto"
                      id="Component_input"
                      onBlur={[Function]}
                      onChange={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      onKeyPress={[Function]}
                      onSelect={[Function]}
                      onTouchStart={[Function]}
                      readOnly={false}
                      spellCheck={true}
                      type="text"
                      value=""
                    />
                  </div>
                  <div
                    className="css-text-901oao r-paddingHorizontal-1e081e0 r-paddingVertical-oyd9sg r-textAlign-q4m81j"
                    dir="auto"
                    style={
                      Object {
                        "WebkitTransform": "translateY(0px)",
                        "color": "rgba(250,108,119,1.00)",
                        "direction": "ltr",
                        "fontFamily": "Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                        "fontSize": "12px",
                        "fontWeight": "400",
                        "height": "18px",
                        "opacity": 0,
                        "paddingBottom": "0px",
                        "paddingLeft": "0px",
                        "paddingTop": "0px",
                        "transform": "translateY(0px)",
                      }
                    }
                  >
                    
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz"
          style={
            Object {
              "WebkitBoxPack": "end",
              "WebkitJustifyContent": "flex-end",
              "justifyContent": "flex-end",
              "msFlexPack": "end",
            }
          }
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1oszu61 r-display-6koalj r-flex-13awgt0 r-justifyContent-1777fci r-padding-t60dpp"
            style={
              Object {
                "backgroundColor": "rgba(0,0,0,0.12)",
                "borderBottomColor": "rgba(0,175,255,1.00)",
                "borderBottomLeftRadius": "50px",
                "borderBottomRightRadius": "50px",
                "borderBottomStyle": "solid",
                "borderBottomWidth": "0px",
                "borderLeftColor": "rgba(0,175,255,1.00)",
                "borderLeftStyle": "solid",
                "borderLeftWidth": "0px",
                "borderRightColor": "rgba(0,175,255,1.00)",
                "borderRightStyle": "solid",
                "borderRightWidth": "0px",
                "borderTopColor": "rgba(0,175,255,1.00)",
                "borderTopLeftRadius": "50px",
                "borderTopRightRadius": "50px",
                "borderTopStyle": "solid",
                "borderTopWidth": "0px",
                "marginBottom": "0px",
                "marginLeft": "0px",
                "marginRight": "0px",
                "marginTop": "0px",
                "minHeight": "44px",
                "minWidth": "64px",
                "paddingLeft": "0px",
                "paddingRight": "0px",
              }
            }
          >
            <div
              aria-disabled={true}
              className="css-view-1dbjc4n r-flex-13awgt0 r-overflow-1udh08x r-position-bnwqim"
              disabled={true}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              role="button"
              style={
                Object {
                  "WebkitAlignSelf": "stretch",
                  "alignSelf": "stretch",
                  "borderBottomLeftRadius": "50px",
                  "borderBottomRightRadius": "50px",
                  "borderTopLeftRadius": "50px",
                  "borderTopRightRadius": "50px",
                  "msFlexItemAlign": "stretch",
                }
              }
            >
              <div
                className="css-view-1dbjc4n r-alignItems-1awozwy r-flex-13awgt0 r-flexDirection-18u37iz r-justifyContent-1777fci"
                style={
                  Object {
                    "WebkitAlignSelf": "stretch",
                    "alignSelf": "stretch",
                    "msFlexItemAlign": "stretch",
                  }
                }
              >
                <div
                  className="css-view-1dbjc4n r-marginHorizontal-1ydw1k6 r-marginVertical-712mwr"
                >
                  <div
                    className="css-view-1dbjc4n r-alignItems-1awozwy r-display-6koalj r-flex-13awgt0 r-flexDirection-18u37iz"
                  >
                    <div
                      className="css-text-901oao r-alignItems-1awozwy r-color-jwli3a r-fontFamily-vnw8o6 r-fontSize-ubezar r-fontWeight-majxgm r-justifyContent-1777fci r-letterSpacing-oxtfae r-lineHeight-hbpseb r-paddingTop-njp1lv r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-tsynxw"
                      dir="auto"
                    >
                      Next
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
